File: 3 - Hermes' Final Word Of Wisdom/solution.go
package main

import (
    "bufio"
    "fmt"
    "os"
    "strings"
)

func findLengthOfLastWord(sentence string) int {
    words := strings.Fields(sentence)
    if len(words) == 0 {
        return 0
    }
    return len(words[len(words)-1])
}

func main() {
    scanner := bufio.NewScanner(os.Stdin)
    scanner.Scan()
    var N int
    fmt.Sscanf(scanner.Text(), "%d", &N)

    for i := 0; i < N; i++ {
        scanner.Scan()
        sentence := scanner.Text()
        fmt.Println(findLengthOfLastWord(sentence))
    }
}

File: 3 - Hermes' Final Word Of Wisdom/Solution.java
import java.util.Scanner;

class Solution{
    private static int findLengthOfLastWord(String sentence) {
        String strings[] = sentence.split(" ");
        if (strings.length == 0) {
            return 0;
        }
        return strings[strings.length - 1].length();
    }
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        scanner.nextLine();
        for (int i = 0; i < N; i++) {
            String sentence = scanner.nextLine();
            System.out.println(findLengthOfLastWord(sentence));
        }
    }
}